webpack/private/webpack_devserver.bzl

```diff
@@ -19,17 +27,51 @@ def webpack_devserver(
     Args:
         name: A unique name for this target.
 
+        srcs: Non-entry point JavaScript source files from the workspace.
+
+            You must not repeat file(s) passed to entry_point/entry_points.
+
+        entry_point: The point where to start the application bundling process.
+
+            See https://webpack.js.org/concepts/entry-points/
+
+            Exactly one of `entry_point` to `entry_points` must be specified.
+
+        entry_points: The map of entry points to bundle names.
+
+            See https://webpack.js.org/concepts/entry-points/
+
+            Exactly one of `entry_point` to `entry_points` must be specified.
+
         webpack_config: Webpack configuration file. See https://webpack.js.org/configuration/.
 
         args: Additional arguments to pass to webpack.
 
             The `serve` command, the webpack config file (`--config`) and the mode (`--mode`) are
             automatically set.
 
+        deps: Runtime dependencies which may be loaded during compilation.
+
+        chdir: Working directory to run Webpack under.
+
+            This is needed to workaround some buggy resolvers in webpack loaders, which assume that the
+            node_modules tree is located in a parent of the working directory rather than a parent of
+            the script with the require statement.
+
+            Note that any relative paths in your configuration may need to be adjusted so they are
+            relative to the new working directory.
+
+            See also:
+            https://github.com/aspect-build/rules_js/blob/main/docs/js_binary.md#js_binary-chdir
+
         data: Runtime dependencies of the program.
 
             The webpack config is automatically passed to data.
```

Comment:
Add to data docstring that it should include the `srcs` and `deps` that would go in `webpack_bundle` in `data` for `webpack_devserver` instead